### PR TITLE
Msw 121225259 twilio error

### PR DESCRIPTION
**Pull request by michaelwehrley**
Original PR: https://github.com/NU-CBITS/think_feel_do_engine/pull/421

### CODE REVIEW

**Comment by ericcf** [comment]
Looks much better to me, thanks!

File: app/models/think_feel_do_engine/reports/sms_delivery_error_record.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module ThinkFeelDoEngine
+  module Reports
+    # Collect metadata for all Participant Emotional Ratings.
```
Comment (eschlange) [inline app/models/think_feel_do_engine/reports/sms_delivery_error_record.rb]: Is this a valid comment?

**Comment by eschlange** [comment]
@michaelwehrley Is the idea behind this change is to: persist Twillio SMS delivery exceptions into a new table and make it available via reports?

**Comment by michaelwehrley** [comment]
@eschlange It is to rescue the Twilio exception while at the same time record numbers that have unsubscribed.

**Comment by eschlange** [comment]
Aside from the comment, everything looks good.

**Pull request merged**